### ISSUE TITLE
Highlight electoral reform in tool for swappers involved in a smaller party swap

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#271]
> It's complicated, always difficult to say where to draw the line. However tactical voting sites that make recommendations in safe seats are usually harmful to the parties who came outside the top two in the seat last time.

> Those recommendations are also harmful to calls for electoral reform. As a minimum they should highlight that tactical voting is unlikely to work there, ideally also pointing people to something like MakeVotesMatter before explaining that small parties benefit from votes in seats like this, even if they lose, so people should vote with their conscience, or seek out a swap with one of @tdg's users.

**Comment by baob** [comment]
@tdg still relevant ?

**Comment by tdg** [comment]
Yes I think so, especially as our org is now going to be proactively working for electoral reform - it could be one opportunity to introduce the movement forward. Whether still relevant in 2029, I hope not (but suspect yes)
